### PR TITLE
Added usage under Powershell to the manual

### PR DESCRIPTION
**Pull request by ferenczy**
Added usage under Powershell and Powershell Core and examples of usage under Unix shells, Powershell, and Windows command shell into the manual.

### CODE REVIEW

**Comment by wtlangford** [comment]
This was added to the 1.6 manual. Can you instead add it to the manual at `docs/content/manual/manual.yml`? Thanks

**Comment by ferenczy** [comment]
@wtlangford Oh, sorry, I missed that file. I'll add it there. Should I also remove it from the 1.6 manual?

**Comment by wtlangford** [comment]
Nah, it's fine to leave it on the 1.6 manual- it'll update on the website the next time I rebuild it, but it won't ever make it to the manpage for 1.6. Having it on the dev manual means it'll be part of the 1.7 release manpage, though

**Comment by ferenczy** [comment]
OK, great. Then I'm done with this pull request, right? Those failed builds are not my fault, I guess.

**Review by itchyny — APPROVED** [review]
LGTM, thank you!

File: docs/content/manual/manual.yml [hunk 1]
```diff
@@ -91,7 +91,15 @@ sections:
...
-      double-quotes in the jq program need backslash escaping.
+      double-quotes in the jq program need backslash escaping. When using
+      the Powershell (`powershell.exe`) or the Powershell Core
+      (`pwsh`/`pwsh.exe`), use single-quote characters around the jq
+      program and backslash-escaped double-quotes (`\"`) inside the jq
+      program.
+
+      Unix shells: `jq '.["foo"]'`
+      Powershell: `jq '.[\"foo\"]'`
+      Windows command shell: `jq ".[\"foo\"]"`
```
Comment (eitsupi) [inline docs/content/manual/manual.yml:102]: Is this correct? On my Windows, both PowerShell 5 and PowerShell 7 seem to work the same as Unix Shell....... `jq '.["foo"]'`

**Pull request merged**